specification/eventgrid/data-plane/Microsoft.Media/stable/2018-01-01/MediaServices.json

```diff
@@ -108,8 +108,837 @@
           "description": "The new state of the Job.",
           "readOnly": true,
           "x-nullable": false
+        },
+        "correlationData": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets the Job correlation data."
         }
       }
+    },
+    "JobError": {
```

Comment:
Will it be possible to prefix "Media" to such types (JobError, JobErrorDetail, JobOutput, etc.). Since autorest didn't fully support associating separate namespaces for each swagger in this set, all of these types will be under Microsoft.Azure.EventGrid.Models namespace, and hence having the Media prefix for your types will make it more easily discoverable and avoid confusion/conflicts.